ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html

```diff
@@ -39,5 +39,6 @@
              data-sly-resource="${paragraph.path @ resourceType=paragraph.resourceType, decorationTagName='div', cssClassName=paragraph.cssClass}"></sly>
         <sly data-sly-resource="${resource.path @ resourceType='wcm/foundation/components/responsivegrid/new', appendPath='/*', decorationTagName='div', cssClassName='new section aem-Grid-newComponent'}"
              data-sly-test="${!isAllowedApplicable && (wcmmode.edit || wcmmode.preview)}"></sly>
+        <sly data-sly-use="${'removeattribute.js' @ referencedPage }"/>
```

Comment:
Can you re-use the constant defined in JAVA class here ? It helps to understand the usage during search

Reply:
FormConstants is present in package `com.adobe.cq.forms.core.components.internal`, here we have keyword `internal` that's why not using in htl file. Please let me know if still it should be added.